apps/dashboard/app/apps/ood_app.rb

```diff
@@ -308,7 +308,18 @@ def passenger_railsdb_app?
 
   # @return [String] memoized version string
   def version
-    @version ||= (version_from_file || version_from_git || "unknown").strip
+    @version ||= if Configuration.Configuration.hide_app_version?
+      nil
+    elsif version_from_file.to_s.downcase == 'unknown'
+      nil
+    else
+      (file_version || version_from_git).strip
```

Comment:
```suggestion
      (version_from_file || version_from_git).strip
```

I don't think this `file_version` exists anymore.